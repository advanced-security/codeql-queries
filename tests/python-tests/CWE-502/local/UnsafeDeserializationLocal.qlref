CWE-502/UnsafeDeserializationLocal.ql