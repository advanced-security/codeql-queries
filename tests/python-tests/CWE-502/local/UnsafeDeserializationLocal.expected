edges
| unsafe.py:5:5:5:11 | ControlFlowNode for input() | unsafe.py:7:14:7:14 | ControlFlowNode for i |
| unsafe.py:10:5:10:32 | ControlFlowNode for Attribute() | unsafe.py:12:14:12:14 | ControlFlowNode for e |
nodes
| unsafe.py:5:5:5:11 | ControlFlowNode for input() | semmle.label | ControlFlowNode for input() |
| unsafe.py:7:14:7:14 | ControlFlowNode for i | semmle.label | ControlFlowNode for i |
| unsafe.py:10:5:10:32 | ControlFlowNode for Attribute() | semmle.label | ControlFlowNode for Attribute() |
| unsafe.py:12:14:12:14 | ControlFlowNode for e | semmle.label | ControlFlowNode for e |
| unsafe.py:17:22:17:29 | ControlFlowNode for Attribute() | semmle.label | ControlFlowNode for Attribute() |
subpaths
#select
| unsafe.py:7:14:7:14 | ControlFlowNode for i | unsafe.py:5:5:5:11 | ControlFlowNode for input() | unsafe.py:7:14:7:14 | ControlFlowNode for i | Deserializing of $@. | unsafe.py:5:5:5:11 | ControlFlowNode for input() | untrusted input |
| unsafe.py:12:14:12:14 | ControlFlowNode for e | unsafe.py:10:5:10:32 | ControlFlowNode for Attribute() | unsafe.py:12:14:12:14 | ControlFlowNode for e | Deserializing of $@. | unsafe.py:10:5:10:32 | ControlFlowNode for Attribute() | untrusted input |
| unsafe.py:17:22:17:29 | ControlFlowNode for Attribute() | unsafe.py:17:22:17:29 | ControlFlowNode for Attribute() | unsafe.py:17:22:17:29 | ControlFlowNode for Attribute() | Deserializing of $@. | unsafe.py:17:22:17:29 | ControlFlowNode for Attribute() | untrusted input |
